### DISCUSSION TITLE
Naming cropped video files automatically from the original video file

### DISCUSSION THREAD

**Discussion by Saheli-R** [bonsai-rx/bonsai#1966] [Q&A]
I have a video file with multiple animal recording and I would like to crop out individual arenas from it, and name it the same as the original file plus add a number to it. I was using the following protocol, repeated for each of the arenas, but it's not working. Could you please help?
![image](https://github.com/user-attachments/assets/be1ec1b4-4dd1-496f-8975-b4812add06a0)
Many thanks,
Saheli

**Comment by bruno-f-cruz** [comment]
Please include the workflow thanks.

**Comment by Saheli-R** [comment]
Hi Bruno,

Please find attached the workflow I was using. Like I mentioned, I have a
video recording 9 flies, and want to crop out each fly individually, and
rename each of the cropped files according to the original filename, plus
add the fly no. to it.
Do let me know where I am going wrong.

Thanks,
Saheli

**Reply by bruno-f-cruz** [reply]
There is no included attachment

**Reply by Saheli-R** [reply]
I attached the .bonsai file via gmail (which is how I had posted the reply) since it says here in Git that this file type is not supported. I am attaching a screenshot here -
![image](https://github.com/user-attachments/assets/26f11fff-20e4-4df9-8289-0828290544a5)

**Reply by bruno-f-cruz** [reply]
Please add the workflow as a zip as suggested in the onboarding post #864 . Thanks

**Reply by Saheli-R** [reply]
[CropFileIndividual.zip](https://github.com/user-attachments/files/16589095/CropFileIndividual.zip)
Hi, sorry, I hadn't gone through the onboarding post earlier. Hope this helps!
